samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AssessmentSettingsBean.java

```diff
@@ -1170,7 +1170,7 @@ public String getStartDateInClientTimezoneString() {
       return this.originalStartDateString;
     }
     else {
-      return userTimeService.dateTimeFormat(startDate, new ResourceLoader().getLocale(), DateFormat.MEDIUM);
+      return userTimeService.dateTimeFormat(startDate, new ResourceLoader().getLocale(), DateFormat.LONG);
```

Comment:
It's not required to create a new ResourceLoader constantly use one that is already declared.

Reply:
this is resolved